app/helpers/grades_helper.rb

```diff
@@ -36,14 +33,68 @@ def has_team_and_metareview?
     {has_team: has_team, has_metareview: has_metareview, true_num: true_num}
   end
 
+  def participant
```

Comment:
This method looks a little bit weird to me. You do not need this method, I think.